Automattic/woocommerce-connect-client
# Remove outer_dimensions when it is empty

This field must be completely removed if it's an empty string. 

The server requires that this field either conform to the `L x W x H` format, or not be included at all. An empty string is not valid.

To test:

First confirm the error with `outer_dimensions`:
1) Try adding a new packages, but be sure to first enter a character in the `outer_dimensions` field, and then delete it. This will result in an empty string getting stored in that field.
2) See that the new package is added to the package list
3) Try saving your package and you should get a server error

Next, repeat those steps on this branch and see that everything works fine.

Also: Try adding a package without a title to see the "Untitled" text that is displayed in that situation.
Also: Try out the clickable icon for the package. It does the same thing as clicking the name.

@allendav @jkudish @jeffstieler @kellychoffman
